packages/babel-parser/src/parser/statement.js

```diff
@@ -1281,10 +1297,7 @@ export default class StatementParser extends ExpressionParser {
       prop.static = false;
       classBody.body.push(this.parseClassProperty(prop));
       return true;
-    } else if (containsEsc) {
```

Comment:
This branch is unreachable because `parseClassMemberFromModifier` is guarded by `isContextual` call, which must return `false` when `containsEsc`.